radar-upload-frontend/README.md

```diff
@@ -0,0 +1,44 @@
+# radar-upload-frontend
+
+## Project setup
+```
```

Comment:
Could you please also add some description of this package to the standard documentation?

Reply:
Could you show me how to do it? And what description are you talking about?